inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -134,18 +134,16 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
     --bslib-dashboard-sidebar-main-bg: var(--#{$prefix}body-bg);
     --bslib-dashboard-main-bg: rgb(247, 247, 247);
     --#{$prefix}card-border-color: var(--bslib-dashboard-border-color-translucent, var(--#{$prefix}border-color-translucent));
-    --bslib-dashboard-card-cap-bg: var(--#{$prefix}body-bg);
     --bslib-dashboard-card-header-font-weight: #{$font-weight-semibold};
-    --bslib-sidebar-bg: var(--#{$prefix}body-bg);
-    --bslib-sidebar-fg: var(--#{$prefix}color);
+    --bslib-sidebar-bg: RGBA(var(--#{$prefix}body-bg-rgb), 0.1);
```

Comment:
Previously we were setting the sidebar background to the body background, but that's an absolute color. 

Now we use the body background color to lighten or darken the sidebar. This still gives us the white sidebar we want for the dashboard design without making the sidebar _always the body bg color_.